python/hail/methods/statgen.py

```diff
@@ -204,6 +204,729 @@ def linreg(dataset, ys, x, covariates=[], root='linreg', block_size=16):
 
     return cleanup(MatrixTable(jm))
 
+@handle_py4j
+@typecheck(dataset=MatrixTable,
+           test=strlike,
+           y=Expression,
+           x=Expression,
+           covariates=listof(Expression),
+           root=strlike)
+def logreg(dataset, test, y, x, covariates=[], root='logreg'):
+    r"""Test each variant for association using logistic regression.
+
+    Examples
+    --------
+
+    Run the logistic regression Wald test per variant using a Boolean
+    phenotype and two covariates stored in column-indexed fields:
+
+    >>> ds_result = methods.logreg(dataset, test='wald', y=dataset.pheno.isCase, x=dataset.GT.num_alt_alleles(), covariates=[dataset.pheno.age, dataset.pheno.isFemale])
```

Comment:
the line wrapping is pretty bad on this line (and on a couple of the tables below). Can you split this over multiple lines?